central/cve/datastore/datastore_impl_test.go

```diff
@@ -28,8 +31,8 @@ import (
 
 var (
 	testSuppressionQuery = searchPkg.NewQueryBuilder().AddBools(searchPkg.CVESuppressed, true).ProtoQuery()
-
-	testAllAccessContext = sac.WithAllAccess(context.Background())
+	ctx, _               = context.WithTimeout(context.Background(), 1*time.Second)
```

Comment:
This should not be a global variable. The timeout clock starts when this variable is initialized, so this should be done inside the function body.